api/models/articles-models.js

```diff
@@ -5,8 +5,39 @@ exports.selectArticleById = (article_id)=> {
     return db.query('SELECT * FROM articles WHERE article_id = $1', [article_id])
     .then(({ rows })=> {
       if(rows.length === 0){
-        return Promise.reject({status: 404, msg: 'Not found'})
+        return Promise.reject({status: 404, msg: 'Article not found'})
       }
       return rows[0];
     })
-  }
\ No newline at end of file
+  }
+
+exports.selectArticles = ()=> {
+  
+  const dbQuery = `
+  SELECT articles.author,
+  articles.title,
+  articles.article_id,
+  articles.topic,
+  articles.created_at,
+  articles.votes,
+  articles.article_img_url,
+  COUNT(comments.comment_id):: integer AS comment_count
+  FROM articles LEFT JOIN comments ON 
+  articles.article_id = comments.article_id
+  GROUP BY 
+  articles.author, 
+  articles.title, 
+  articles.article_id, 
+  articles.topic, 
+  articles.created_at, 
+  articles.votes, 
+  articles.article_img_url
+  ORDER BY articles.created_at DESC
+  `
```

Comment:
The GROUP BY seems a little verbose and I wonder whether it is working as expected. We want to count the comments that have the same article_id so you can probably just GROUP BY that